setup.cfg

```diff
@@ -62,13 +62,6 @@ dev =
     ## Constrain system
     pylint
 
-   
-    
-[semantic_release]
-version_variable=trestle_fedramp/__init__.py:__version__
-branch = main
-upload_to_pypi = true
-version_source = commit
 
```

Comment:
Don't we need this? I believe it is there in trestle.

Reply:
`setup.cfg` is not support by the version of [`semantic-release`](https://python-semantic-release.readthedocs.io/en/latest/migrating_from_v7.html#setup-cfg-is-no-longer-supported) we are using so I do not believe it is being read in `trestle`. However, tagging @butler54 @AleJo2995 just in case I misunderstood.